demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -729,53 +734,73 @@ static void otaAppCallback( OtaJobEvent_t event,
 {
     OtaErr_t err = OtaErrUninitialized;
 
-    /* OTA job is completed. so delete the MQTT and network connection. */
-    if( event == OtaJobEventActivate )
+    switch( event )
     {
-        LogInfo( ( "Received OtaJobEventActivate callback from OTA Agent." ) );
+        case OtaJobEventActivate:
+            LogInfo( ( "Received OtaJobEventActivate callback from OTA Agent." ) );
 
-        /* Activate the new firmware image. */
-        OTA_ActivateNewImage();
+            /* Activate the new firmware image. */
+            OTA_ActivateNewImage();
 
-        /* Shutdown OTA Agent. */
-        OTA_Shutdown( 0 );
+            /* Shutdown OTA Agent. */
+            OTA_Shutdown( 0 );
 
-        /* Requires manual activation of new image.*/
-        LogError( ( "New image activation failed." ) );
-    }
-    else if( event == OtaJobEventFail )
-    {
-        LogInfo( ( "Received OtaJobEventFail callback from OTA Agent." ) );
+            /* Requires manual activation of new image.*/
+            LogError( ( "New image activation failed." ) );
```

Comment:
Would it make sense to add to this log that the user should manually activate the image? As a user, I would see this in the console and think something went wrong.

Reply:
I see, that makes sense. Thanks.